vhdbuilder/packer/components.json

```diff
@@ -327,11 +324,9 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/open-policy-agent/gatekeeper:*",
       "amd64OnlyVersions": [
-        "v3.4.1",
-        "v3.5.2"
```

Comment:
v3.5.2 and v3.7.0 was not used in RP for a while.

{{- else if eq .component "open-policy-agent-gatekeeper" -}}
  {{- if semverCompare ">=1.18.0" .version -}}v3.7.1
  {{- else -}}v3.4.1
  {{- end -}}